base_local_planner/include/base_local_planner/obstacle_cost_function.h

```diff
@@ -61,13 +61,13 @@ class ObstacleCostFunction : public TrajectoryCostFunction {
 
   void setSumScores(bool score_sums){ sum_scores_=score_sums; }
 
-  void setParams(double max_trans_vel, double max_forward_inflation, double max_sideward_inflation, double scaling_speed);
+  void setParams(double max_trans_vel, double max_forward_inflation, double max_sideward_inflation, double scaling_speed, bool occdist_use_footprint = true);
```

Comment:
S: I guess this is never called w/o `occdist_use_footprint`, so no need default 
I meant in the dyn reconfigure

Reply:
I know, I agree with you :sob: But @ayushgaud forced me to do this :sob: